### ISSUE TITLE
[Question]:MoCreatures' horses can drop horse meat from Improving Minecraft mods

### ISSUE THREAD

**Issue by Omgise** [Rozmir-Rohi/mo-creatures-legacy#15]

**Comment by Rozmir-Rohi** [comment]
I can try to add this in.
What mods add horse meat?

**Comment by Omgise** [comment]
https://github.com/GregTech6/gregtech6
https://github.com/Blood-Asp/GT5-Unofficial

**Comment by Rozmir-Rohi** [comment]
Would you like horse meat from all these mods integrated?

If you would like more than one of these mods to be integrated, in what order would you like them to be prioritized?

**Comment by Rozmir-Rohi** [comment]
Just to confirm the order is:
1. GregTech 6
2. GegTech 5
3. Terrafirmacraft+
4. Terrafirmacraft
5. Food Expansion
6. Improving Minecraft

So if all these mods are installed, MoC Horses will drop horse meat from GregTech 6

**Comment by Omgise** [comment]
Yes, MoC Horses will drop horse meat from GregTech 6

**Comment by Rozmir-Rohi** [comment]
ok, I can try to add this in.

**Comment by Rozmir-Rohi** [comment]
I found out that GregTech 6 integrates Mo'Creatures horses for horse meat drops by default, and since I didn't change the entity IDs from the original Mo' Creatures this feature from GregTech 6 still works.

Also, I found that GregTech 5 doesn't have a horse meat item so I can't add horse meat drops from GregTech 5.

For Terrafirmacraft+ and Terrafirmacraft, both of them change the health of vanilla entities to be around 1000hp, so to make Mo' Creatures entities fit into Terrafirmacraft I would have to add code for all Mo'Creatures entities to change their health when Terrafirmacraft is loaded. Also, to my knowledge, Terrafirmacraft+ and Terrafirmacraft completely replace the normal spawning system, so I'm not sure if they will even allow Mo'Creatures entities to spawn in worlds. If you know a way to make Terrafirmacraft worlds allow the spawning of Mo'Creatures entities, I might still add integration with Terrafirmacraft+ and Terrafirmacraft.


Because of the reasons above, I will be adding horse meat integration for:
1. Food Expansion
2. Improving Minecraft
